assets/js/src/core/modules/element/actions/delete/use-delete.tsx

```diff
@@ -146,6 +150,12 @@ export const useDelete = (elementType: ElementType): UseDeleteHookReturn => {
       }))
     } else if (parentId !== undefined) {
       refreshTree(parentId)
+
+      dispatch(
```

Comment:
Now as we restructed the `useElementRefresh` it would be possible to use it here, right?

Reply:
done with https://github.com/pimcore/studio-ui-bundle/pull/884/commits/28155f588755b49eebb7ed9d1b3f8fec31c3ced1 and https://github.com/pimcore/studio-ui-bundle/pull/884/commits/6ded9e78767aa2f4ae93ff4d7ba8cb569597f4c2